editor/src/components/editor/npm-dependency/npm-dependency.ts

```diff
@@ -324,19 +370,32 @@ export function dependenciesFromPackageJson(
   }
 }
 
+export function dependenciesFromPackageJson(
+  packageJsonFile: ProjectFile | null,
+  includeDevDependencies: boolean,
```

Comment:
in my fight against boolean flags, can we make this something like:

```suggestion
  includeDevDependencies: 'dependencies' | 'combined',
```

or

```suggestion
  includeDevDependencies: 'productionDependencies' | 'combined',
```

Reply:
Yep good shout.

Reply:
Done